x/incentives/genesis.go

```diff
@@ -3,6 +3,8 @@ package incentives
 import (
 	"sort"
 
+	"cosmossdk.io/errors"
```

Comment:
Nit: refactor to be consistent with codebase
```suggestion
	errorsmod "cosmossdk.io/errors"
```